requests/api.py

```diff
@@ -123,7 +141,6 @@ def put(url, data=None, **kwargs):
     :param url: URL for the new :class:`Request` object.
     :param data: (optional) Dictionary, list of tuples, bytes, or file-like
         object to send in the body of the :class:`Request`.
-    :param json: (optional) json data to send in the body of the :class:`Request`.
```

Comment:
The alternative would have been to introduce `json` as positional argument, but as it is currently neither in `requests.api` nor `requests.sessions` I just updated the doc string.